### DISCUSSION TITLE
Share whole nested documents/collections

### DISCUSSION THREAD

**Discussion by bihlmaier** [outline/outline#2010] [Ideas]
It would be nice to add the option to share a whole collection, including nested documents.

It is possible to share each document separately, but this can be very time-consuming.
Another problem is the linking to other documents.

If I add linkings to another document (which is also shared), It will link internally to the document.
So navigating through the documents with the shared links is nearly impossible.

### So my feature request is the following:

- share collection/nested document (with an extra option like "include sharing of all subordinate documents inside this collection).
- display the subordinate documents from the superior documents
- shared documents automatically replace internal links to shared links

What do you think about my idea?

**Comment by tommoor** [comment]
Yes, these are desired features – please follow the issues here:

- https://github.com/outline/outline/issues/1141
- https://github.com/outline/outline/issues/1160

Could I ask what is your particular usecase?

**Reply by Abdillah** [reply]
I would like to see "include sharing for all subordinates" that would be a great feature.
Though, I started a new discussion here: https://github.com/outline/outline/discussions/3587
